view/template/layout/_header_analytics.php

```diff
@@ -0,0 +1,10 @@
+<?php if (IS_PRODUCTION): ?>
```

Comment:
I don't see a strong reason to put this in `<head>`. Just leave it where it is, unless I'm misunderstanding and it doesn't work in the footer (it should).